src/components/CreateAccountPage.js

```diff
@@ -0,0 +1,106 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import { Button, FormControl, FormGroup } from 'react-bootstrap'
+import { Link } from 'react-router-dom'
+
+/* global fetch */
+
+export default class LogInPage extends Component {
+  constructor (props) {
+    super(props)
+
+    this.getValidationState = this.getValidationState.bind(this)
+    this.checkValidationState = this.checkValidationState.bind(this)
+    this.handleLogInClick = this.handleLogInClick.bind(this)
+    this.createAccount = this.createAccount.bind(this)
+  }
+
+  getValidationState () {
+    let checkInfo = this.props.accountName
+
+    let found = Object.keys(this.props.dbAccounts).findIndex((name) => {
+      return name === checkInfo
+    })
+
+    if (found !== -1 || checkInfo.length < 3) {
+      return 'warning'
```

Comment:
I prefer to see magic strings like this defined as a constant at the top of the file, e.g.:
```
const WARNING_INDICATOR = 'warning'
const SUCCESS_INDICATOR = 'success'
...
if (found !== -1 || checkInfo.length <3) {
  return WARNING_INDICATOR
...
```
This is because of the following reasons:
1. the variable name provides some level of documentation for what the string is for
2. if you make an error inside the string, 'warning', and accidentally type 'wanring', the linter won't catch it. But if you accidentally type `WANRING_INDICATOR`, the linter will catch it (and you have linting tied into your editor right?) which makes catching errors much quicker.
3. By defining the constants in one place and importing them rather that just typing the magic string in, you're less likely to accidentally use incorrect data across files (e.g. you could easily see a situation where in one file you use `success` and in the other file you accidentally use `successful`), this type of error will be caught immediately.